pwnagotchi/locale/de/LC_MESSAGES/voice.po

```diff
@@ -203,11 +203,11 @@ msgstr "Ops, da ist was schief gelaufen... Starte neu..."
 
 #, python-brace-format
 msgid "Kicked {num} stations\n"
-msgstr "{num} Stationen gekicked\n"
+msgstr "{num} Stationen gekickt\n"
 
 #, python-brace-format
 msgid "Made {num} new friends\n"
-msgstr "{num} Freunde gefunden\n"
+msgstr "{num} neue Freunde gefunden\n"
```

Comment:
should `Freunde` be capitalised like it is here?

Reply:
yes, any nouns get capitalized in German, not just proper nouns